snippets/fsh-snippets.json

```diff
@@ -0,0 +1,63 @@
+{
+	"FSH Profile": {
+		"prefix": "pro",
+		"body": [
+			"Profile: $1",
+			"Parent: $2",
+			"Id: ${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}",
+			"Title: \"${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}\"",
+			"Description: \"$5\"",
+			"* $0"
+		],
+		"description": "Create a FSH Profile"
+	},
+
+	"FSH Extension": {
+		"prefix": "ext",
+		"body": [
+			"Extension: $1",
+			"Id: ${1/(([A-Z]*)[_]([A-Z]*))|(([A-Z]*[a-z0-9]+)([A-Z]+))|(([A-Z]+)([A-Z0-9])([a-z]))/${2:/downcase}${5:/downcase}${8:/downcase}-${3:/downcase}${6:/downcase}${9:/downcase}$10/g}",
+			"Title: \"${1/[_]|(([a-z0-9])([A-Z]))|(([A-Z])([A-Z0-9])([a-z]))/$2$5 $3$6$7/g}\"",
+			"Description: \"$2\"",
+			"* $0"
+		],
+		"description": "Create a FSH Extension"
+	},
+
+	"FSH Value Set": {
+		"prefix": "vs",
```

Comment:
It looks like the `prefix` can be an array of values. What would you think of potentially having "vs" and "val" trigger this snippet? I'm thinking that it might be nice to have it come up as a suggestion as you start to type out the "ValueSet" keyword, similar to how "pro" triggers "Profile". That way you wouldn't need to necessarily know you want to use the snippet when you start creating the FSH item.

Reply:
Good idea, @jafeltra -- updated in 53e124c.